packages/ui/quantity/Quantity.tsx

```diff
@@ -0,0 +1,27 @@
+import { useCounter } from '@mantine/hooks';
+
+import {
+  BsDashStyled,
+  BsPlusStyled,
+  Count,
+  QuantityWrapper,
+  SquareButton,
+} from './style';
+
+export function Quantity() {
+  const [count, handlers] = useCounter(0, { min: 1, max: 100 });
+
+  return (
+    <QuantityWrapper>
+      <SquareButton onClick={handlers.decrement}>
+        <BsDashStyled />
+      </SquareButton>
+      <SquareButton>
+        <Count>{count}</Count>
```

Comment:
Allow input of number for quantity

Reply:
also handle number only input for this change

Reply:
done